imap_processing/codice/L0/decom_python_example.py

```diff
@@ -0,0 +1,218 @@
+"""
+This script contains functions to extract and decommute parameters from a binary file containing packets.
+This is a hard coded version of the decom.py file. It is used to test the decom.py file.
```

Comment:
```suggestion
This is a hard coded version of the `decom.py` file. It is used to test the `decom.py` file.
```

Apologies if you are already aware of this -- I just wanted to use this suggestion to point out that if you surround certain keywords with "`", then they will render as monospace format in sphinx/ReadTheDocs, which looks cool!

Typically developers use this to format special keywords or filenames in their documentation, e.g.

```
We used the `spiceypy` library to create the file `my_spice_kernel.txt`
```

[This page](https://imap-processing.readthedocs.io/en/latest/development/style_guide/python-coding.html) shows a good example of this. Note how the `black`, `temperature`, `filename` are rendered differently.